docs/sphinx/examples/cpp/basics/noise_depolarization.cpp

```diff
@@ -12,18 +12,18 @@
 // CUDA Quantum supports several different models of noise. In this
 // case, we will examine the modeling of depolarization noise. This
 // depolarization will result in the qubit state decaying into a mix
-// of the basis states, |0> and |1>, with a user provided probability.
+// of the basis states, |0> and |1>, with a user-provided probability.
 
 int main() {
 
   // We will begin by defining an empty noise model that we will add
   // our depolarization channel to.
   cudaq::noise_model noise;
 
-  // Depolarization channel with `1.0` probability of the qubit state
-  // being scrambled.
+  // We define a depolarization channel setting to `1.0` the probability of the
+  // qubit state being scrambled.
```

Comment:
similar, moving 1.0 to the end of the sentence

Reply:
+1

Reply:
the last commit fixes this.